src/Type.cc

```diff
@@ -1337,6 +1337,10 @@ void RecordType::Create(std::vector<std::optional<ZVal>>& r) const
 		{
 		auto& init = field_inits[i];
```

Comment:
Could we drop the `&` here and possibly make it `auto*` although that seems to be deduced. Minimally with the null check below that was a bit confusing.